website/cue/reference/components/sources/host_metrics.cue

```diff
@@ -116,6 +117,72 @@ components: sources: host_metrics: {
 				unit:    "seconds"
 			}
 		}
+		cgroups: {
+			common:      false
+			description: #"Options for the "cgroups" metrics collector. This collector is only available on Linux systems, as the cgroups (control groups) system is only present on Linux, and only supports either version 2 or hybrid cgroups."#
```

Comment:
```suggestion
			description: #"""
			Options for the "cgroups" metrics collector.
			
			Note: this collector is only available on Linux systems and only supports either version 2 or hybrid cgroups.
			"""#
```

I think this is slightly easier to read.

Reply:
Applied in 0a55672e7